env-example

```diff
@@ -69,7 +73,9 @@ COMPOSE_CONVERT_WINDOWS_PATHS=1
 ### Environment ###########################################
 
 # If you need to change the sources (i.e. to China), set CHANGE_SOURCE to true
-CHANGE_SOURCE=false
+
+CHANGE_SOURCE=true
+
```

Comment:
This adjustment should be to modify .env

Reply:
got it